package/iframe/require.js

```diff
@@ -0,0 +1,961 @@
+/**
```

Comment:
maybe call this file ```bundle.js``` (to not be confused with r.js), and possibly gitignore this file